src/pages/messages/messages.ts

```diff
@@ -141,13 +141,19 @@ export class MessagesPage implements OnInit, OnDestroy {
   // Removes the scroll listener once all messages from the past were fetched
   autoRemoveScrollListener<T>(messagesCount: number): Observable<T> {
     return Observable.create((observer: Subscriber<T>) => {
-      Messages.find().subscribe((messages) => {
-        // Once all messages have been fetched
-        if (messagesCount != messages.length) return;
-        // Signal to stop listening to the scroll event
-        observer.next();
-        // Finish the observation to prevent unnecessary calculations
-        observer.complete();
+      Messages.find().subscribe({
```

Comment:
The observable already existed here, I only added an `error` callback in case it fails. This observable is used as a limitation for the `scroll` event observable. Each time we scroll to the top of the `messages` page, new messages are being fetched, and once we fetched all of them, we want to stop the auto-fetching. Of course, we should add this note once composing the appropriate step in the tutorial.